cloudsync/command/list.py

```diff
@@ -0,0 +1,60 @@
+import os
+import logging
+import datetime
+
+from typing import Union
+
+from cloudsync.utils import debug_sig
+
+from .utils import CloudURI, SubCmd
+
+# all files written as user-only
+os.umask(0o77)
```

Comment:
This umask should only affect the config folder, not the files being synced

Reply:
cleaned it up